pkg/cmd/docgen/diagrams.go

```diff
@@ -677,8 +677,20 @@ var specs = []stmtSpec{
 		inline: []string{"explain_option_list"},
 		replace: map[string]string{
 			"explain_option_name": "( 'VERBOSE' | 'TYPES' | 'OPT' | 'DISTSQL' )",
+			"'ANALYZE'":           "",
```

Comment:
Instead of a replace rule, you should use an exclude rule here which will prevent the double identical lines:

`exclude: []*regexp.Regexp{regexp.MustCompile("'ANALYZE'")},`